README.md

```diff
@@ -70,4 +70,4 @@ Filter todos by status `All` / `Active` / `Completed`:
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-loading-todos/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://natamrshn.github.io/react_todo-app-loading-todos/) and add it to the PR description.
```

Comment:
Add the demo link to the PR description 
<img width="944" alt="image" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/64750958/bb64b00c-f7f7-4a7e-903b-5aa25c8dcc6d">

Reply:
steel need it